gears/src/types/query/account.rs

```diff
@@ -13,27 +12,22 @@ mod inner {
 #[derive(Clone, PartialEq, Serialize, Deserialize, Debug)]
 pub struct QueryAccountResponse {
     /// account defines the account of the corresponding address.
-    pub account: Account,
+    pub account: Option<Account>,
```

Comment:
What a reason for these changes? Do we have a case where `None` is acceptable?

Reply:
Yes, if you query for an account that doesn't exist.

Reply:
Okay. I missed that this is Response struct